src/config/config.ts

```diff
@@ -94,6 +94,17 @@ function applyDefaultConfigurationValues(serverConfig: any) {
             }
         }
     });
+
+    // This pass handles boolean values that aren't in ServerConfigDefaults
+    // Without this, these boolean properties will be rendered as string
+    Object.keys(serverConfig).forEach(key => {
```

Comment:
@arishta-dev might be nice to console a warning message when we encounter such a case because we want to add missing items to the defaults.

Reply:
Done